crates/uv/tests/pip_compile.rs

```diff
@@ -6980,7 +6980,7 @@ fn universal_no_repeated_unconditional_distributions() -> Result<()> {
         # via sphinx
     tomli==2.0.1 ; python_version < '3.11'
         # via pylint
-    tomlkit==0.12.4
+    tomlkit==0.12.4 ; python_version < '3.11' or python_version >= '3.12'
```

Comment:
I am unsure if this new snapshot is correct. The marker expressions look awfully suspicious here.